safenode/src/protocol/mod.rs

```diff
@@ -12,3 +12,100 @@ pub mod error;
 pub mod messages;
 /// Storage types for spends, chunks and registers.
 pub mod storage;
+
+use self::storage::{ChunkAddress, DbcAddress, RegisterAddress};
+
+use libp2p::{
```

Comment:
We shouldn't have dependencies to crates which are implementation of the `protocol`, this mod/crate doesn't and shouldn't depend on any SAFE crate or crates needed for implementation o the protocol.